### PR TITLE
chore(deps): upgrade dependencies - EUBFR-164

### PR DESCRIPTION
**Pull request by kalinchernev**
# PR description

**Use `dev2` in order to test in isolation from `dev`.**
All functionalities should work as before.

File size of lambda functions is preserved, no loss:

lambda management console

### CODE REVIEW

File: services/enrichment/manager/webpack.config.js [hunk 1]
```diff
@@ -4,19 +4,28 @@ const path = require('path');
 module.exports = {
   entry: slsw.lib.entries,
   target: 'node',
+  mode: slsw.lib.webpack.isLocal ? 'development' : 'production',
+  optimization: {
+    minimize: false,
```
Comment (yhuard) [inline services/enrichment/manager/webpack.config.js]: In prod, you'll probably want to optimize the code

**Review by yhuard — COMMENTED** [review]
overall it looks okay, though i haven't tested it yet

File: services/enrichment/manager/webpack.config.js (hunk 1 above)
Comment (kalinchernev) [reply services/enrichment/manager/webpack.config.js]: Honestly, I did initially. The minimization didn't really result in gain of less size and then I remembered that we jumped to node 8 mainly to be able to read async/await easily if something breaks. And long story short, I kept it like this for potentially read code in lambda functions when needed, but i'll change to optimize and we can un-optimize when needed.

**Comment by yhuard** [comment]
it starts with `./scripts/getECL.sh: line 9: wget: command not found` 😆

**Comment by yhuard** [comment]
I will update the ECL to make sure `dist` are attached to the presets packages on npm

**Review by yhuard — APPROVED** [review]

**Pull request merged**